twake/backend/node/src/cli/cmds/generate_cmds/users-and-companies.ts

```diff
@@ -57,7 +57,7 @@ const command: yargs.CommandModule<{}, CLIArgs> = {
       console.log("Creating user", userInCompany);
       const created = await userService.users.create(getUserInstance(userInCompany.user));
 
-      return userService.companies.addUserInCompany(userInCompany.company, created.entity);
+      return userService.companies.setUserRole(userInCompany.company.id, created.entity.id);
```

Comment:
Can we remove a user from a company with this `setUserRole` method ?

Reply:
We have `removeUserFromCompany` function for that

Reply:
So "setUserRole" automatically add the user in the company if it doesn't exists ?

Reply:
yes it is. 
I can rename it to get away from ambiguity

Reply:
No it's great don't change it :)